python/ray/experimental/sgd/pytorch/pytorch_trainer.py

```diff
@@ -211,12 +263,30 @@ def restore(self, checkpoint):
         state_id = ray.put(state)
         ray.get([worker.set_state.remote(state_id) for worker in self.workers])
 
-    def shutdown(self):
+    def shutdown(self, force=False):
         """Shuts down workers and releases resources."""
         for worker in self.workers:
-            worker.shutdown.remote()
+            if not force:
+                worker.shutdown.remote()
```

Comment:
do we need to `ray.get(worker.shutdown.remote())`?

Reply:
no need I think...